packages/medusa/src/api/routes/store/products/list-products.ts

```diff
@@ -312,6 +331,170 @@ export default async (req, res) => {
   })
 }
 
+async function listAndCountProductWithIsolatedProductModule(
+  req,
+  filterableFields,
+  listConfig
+) {
+  // TODO: Add support for fields/expands
```

Comment:
comment: I would not focus on adding this in right now

Reply:
Yes I did not plan to include it in the first iteration. I think we can wait a bit for that to stabilizing the API